policies/trademark-policy.md

```diff
@@ -1,139 +1,122 @@
-Your right to use a mark of the Commonhaus Foundation is provided for in this policy and in the statement of permitted use, if any, that may accompany the trademark notice displayed on a CF project website or described in the project's `TRADEMARK.md` file.
+## Guidelines for Commonhaus Trademarks
 
-A copyright license, even an open source copyright license, does not include an implied right or license to use a trademark that may be related to the project developing the licensed software or other materials.
-Because open source licenses permit unrestricted modification of the copyrighted software, and the Commonhaus Foundation has no intention of interfering with that right, your right to use any specific trademark of the Commonhaus Foundation is not determined by your use of software made available under an open source license.
-Your right to use a mark is conditioned upon conforming to other requirements that are directly related to the qualities that the specific mark has been created to preserve.
+The use of Commonhaus Foundation (CF) Marks is governed by the following detailed guidelines. These are designed to ensure clarity, prevent misuse, and maintain the integrity of the Marks.
 
-Proper use of Commonhaus Foundation trademarks by following these trademark usage guidelines protects the value of Commonhaus Foundation trademarks.
-Any use of or reference to Commonhaus trademarks that is inconsistent with this trademark policy, or use of marks that are confusingly similar to trademarks of the Commonhaus Foundation, is prohibited.
-All uses of Commonhaus Foundation trademarks, and all goodwill associated therewith, will inure solely to the benefit of the Commonhaus Foundation, and Commonhaus Foundation projects.
+### Acknowledgment and Symbols
 
-## Guidelines for Commonhaus Trademarks
+- Use the CF Marks in a form that distinguishes them from the text around them, such as by capitalization, bold or italic fonts, or with quotation marks.
 
-Understanding and respecting trademark rules are vital for preserving the integrity and recognition of a brand. These guidelines outline how to correctly use any trademark, especially those associated with the Commonhaus Foundation.
+- Use the appropriate symbol (® for registered trademarks; ™ for unregistered; ℠ for unregistered service marks) next to the first use of CF Marks. For a detailed list, visit [Commonhaus Trademarks](https://www.commonhaus.org/trademarks/).
 
-### General Trademark Rules
+- Use the full form of any trademarks (e.g., Commonhaus Foundation”) in the first reference in all documents of mass communication, including marketing collateral and web pages. You may then use any abbreviated or short form references (e.g. CF) within the same document where the full form has already been used.
 
-- **Nature of Use:**
-    - Trademarks should be used as adjectives followed by a generic noun, not as verbs or nouns.
-    - Avoid using trademarks in plural or possessive forms.
```

Comment:
This looks a bit silly to me. Everyone I know uses "Hibernate" as a noun _all the time_, and phrases such as "Hibernate's approach to xxxx" are very common.

Reply:
Will ask